main.py

```diff
@@ -2,6 +2,7 @@
 
 from bookstore import Book, BookStore
 from menu import Menu
+from readinglist.bookstore import BookError
```

Comment:
Not sure what this line is for. When I tried testing the code, this line caused an immediate crash. I commented it out and everything else worked fine afterwards. Besides this causing a crash the changes are good.